src/common/snippets/src/pass/split_dimension_m.cpp

```diff
@@ -217,9 +217,13 @@ void SplitDimensionM::reshape_subgraph(const std::shared_ptr<op::Subgraph>& subg
             return;
 
         const auto shape = param->get_partial_shape().get_shape();
+        // if Shape rank is less than index of dimension M, no need to reshape it.
+        if (shape.size() <= dim_M_index)
```

Comment:
Comment and condition mismatch: shape rank is "less" or "less or equal"?

Reply:
Rephrase. Please take a look again!